test/TokenVault.test.ts

```diff
@@ -25,52 +21,52 @@ describe('Vault', () => {
+            expect(await tokenVault.approved(wallet2.address)).to.equal(true)
         })
     })
 
     describe('increaseERC20Allowance()', () => {
         it('should allow owners to increase approval amount', async () => {
-            await vault.setApproved(wallet2.address, true)
-            await token.mock.allowance.withArgs(vault.address, wallet2.address).returns('0')
+            await tokenVault.setApproved(wallet2.address, true)
+            await token.mock.allowance.withArgs(tokenVault.address, wallet2.address).returns('0')
             await token.mock.approve.withArgs(wallet2.address, '1111').returns(true)
-            await vault.increaseERC20Allowance(token.address, wallet2.address, '1111')
+            await tokenVault.increaseERC20Allowance(token.address, wallet2.address, '1111')
         })
 
         it('should allow managers to increase approval amount', async () => {
-            await vault.setManager(wallet3.address)
-            await vault.setApproved(wallet2.address, true)
-            await token.mock.allowance.withArgs(vault.address, wallet2.address).returns('0')
+            await tokenVault.setManager(wallet3.address)
+            await tokenVault.setApproved(wallet2.address, true)
+            await token.mock.allowance.withArgs(tokenVault.address, wallet2.address).returns('0')
             await token.mock.approve.withArgs(wallet2.address, '1111').returns(true)
-            await vault.connect(wallet3).increaseERC20Allowance(token.address, wallet2.address, '1111')
+            await tokenVault.connect(wallet3).increaseERC20Allowance(token.address, wallet2.address, '1111')
         })
     })
 
     describe('decreaseERC20Allowance()', () => {
         it('should allow manager to decrease approval amount', async () => {
-            await token.mock.allowance.withArgs(vault.address, wallet2.address).returns('1111')
+            await token.mock.allowance.withArgs(tokenVault.address, wallet2.address).returns('1111')
             await token.mock.approve.withArgs(wallet2.address, '111').returns(true)
-            await vault.decreaseERC20Allowance(token.address, wallet2.address, '1000')
+            await tokenVault.decreaseERC20Allowance(token.address, wallet2.address, '1000')
         })
 
         it('should allow manager to decrease approval amount', async () => {
-            await vault.setManager(wallet3.address)
-            await token.mock.allowance.withArgs(vault.address, wallet2.address).returns('1111')
+            await tokenVault.setManager(wallet3.address)
+            await token.mock.allowance.withArgs(tokenVault.address, wallet2.address).returns('1111')
             await token.mock.approve.withArgs(wallet2.address, '0').returns(true)
-            await vault.connect(wallet3).decreaseERC20Allowance(token.address, wallet2.address, '1111')
+            await tokenVault.connect(wallet3).decreaseERC20Allowance(token.address, wallet2.address, '1111')
```

Comment:
and here

Reply:
Done: https://github.com/pooltogether/v4-core/pull/272/commits/b9457454673816223dca65caaf45b6fa23e2b5a0